website/docs/docs/cloud/connect-data-platform/connect-onehouse.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Connect Onehouse"
+id: connect-onehouse
+description: "Setup instructions for connecting Onehouse to dbt Cloud"
+sidebar_label: "Connect Onehouse"
+pagination_next: null
+---
+
+<Snippet path="dbt-onehouse" />
+
+<Constant name="cloud" /> Supports connecting to [Onehouse SQL](https://www.onehouse.ai/) using the Apache Spark Connector with the Thrift method.
+
+**Connect to a Onehouse SQL Cluster with the `dbt-spark` adapter.**
+
+## Requirements
+
+* For dbt Cloud, ensure your Onehouse SQL endpoint is accessible via external DNS/IP, whitelisting dbt Cloud IPs.
+* For dbt Core, you can SSH tunnel into the VPC to access the endpoint, without opening up ports.
+
+## What works 
+
+* All dbt features like `dbt clean`, `dbt compile`, `dbt debug`, `dbt seed`, and `dbt run`.
+* dbt materialized type of "table" and "incremental" 
+* Apache Hudi table types of Merge on Read (MoR) and Copy on Write (CoW). It is recommended to use MoR for mutable workloads.
+
+## Limitations
+
+* dbt materialized type cannot be "view".
+* `dbt seed` has row / record limits.
+* `dbt seed` only supports CoW tables.
+
+## dbt connection
+
+Fill in the following fields when creating an **Apache Spark** warehouse connection using the Thrift connection method:
+
+| Field | Description | Examples |
+| ----- | ----------- | -------- |
+| Method | The method for connecting to Spark | Thrift |
+| Hostname | The hostname of your Onehouse SQL Cluster endpoint | `yourProject.sparkHost.com` |
+| Port | The port to connect to Spark on | 10000 |
+| Cluster | Onehouse does not use this field | |
+| Connection Timeout | Number of seconds after which to timeout a connection | 10 |
+| Connection Retries | Number of times to attempt connecting to cluster before failing | 0 |
+| Organization | Onehouse does not use this field | |
+| User | Optional. Not enabled by default. | dbt_cloud_user |
+| Auth | Optional, supply if using Kerberos. Not enabled by default. | `KERBEROS` |
+| Kerberos Service Name | Optional, supply if using Kerberos. Not enabled by default. | `hive` |
+
+<Lightbox src="/img/onehouse/onehouse-dbt.png" width="70%" title="Onehouse configuration"/>
+
+## dbt project
+
+When using dbt, ensure you add necessary configurations to dbt_project.yml for the dbt connector to write data correctly.
```

Comment:
I think you should call out that this is needed for the project to default to this. But you can override them on the resources.yml or model config

Reply:
it's different for different users especially the location_root.

Reply:
I don't understand your response? By putting it in the `dbt_project.yml` you are making this the default setting for the project. I just want to make sure users know that's what they're doing